portalnet/src/storage.rs

```diff
@@ -1073,15 +1079,12 @@ pub mod test {
         let storage_config = PortalStorageConfig::new(min_capacity, node_id, false).unwrap();
         let mut storage = PortalStorage::new(storage_config.clone(), ProtocolId::History)?;
 
-        // Fill up the storage. This is overkill for the 1kb capacity, but an upcoming
-        // change will make the minimum storage size 1MB, so 32 keys should still be sufficient then.
+        // Fill up the storage.
         for _ in 0..32 {
             let content_key = generate_random_content_key();
-            let value: Vec<u8> = "OGFWs179fWnqmjvHQFGHszXloc3Wzdb4".into();
+            let value: Vec<u8> = vec![0; 32000];
             storage.store(&content_key, &value)?;
             // Speed up the test by ending the loop as soon as possible
-            // At 1kb, that should be immediately after the first item is stored.
-            // At 1MB, it will take the full 32 items.
```

Comment:
lol, why did I write that 32 items at 32 bytes would fill 1MB? 😅 Just off by a factor of 1000x :P

Thanks for deleting this garbage.